solr/conf/schema.xml

```diff
@@ -187,4 +194,7 @@
   <copyField source="dct_provenance_s" dest="suggest"/>
   <copyField source="dc_subject_sm" dest="suggest"/>
   <copyField source="dct_spatial_sm" dest="suggest"/>
-</schema>
+  
+  <!-- for bbox value -->
+  <copyField source="*_geom" dest="*_bboxtype"/>
```

Comment:
Can we assume that all `*_geom` fields can be copied into `_*bboxtype`? If I provide point data here, what happens?

Reply:
I will try to test that out since that is something we should evaluate.  If this copying over doesn't work for point fields, don't know if there is such a thing as a conditional copy possible in schema (i.e. If solr_geom is bbox-compatible, copy over)

Reply:
Out current schema uses the "ENVELOPE WKT" syntax in the `solr_geom` field which will copy over fine. Instead of copying all `_geom` fields, how about just `solr_geom`?

Reply:
@hudajkhan @mejackreed @eliotjordan We are all on the same page here. Let's limit the copy to just the solr_geom field. Want to push that commit @hudajkhan?

Reply:
Will make this change now.